cicd/internal/flags/common-flags.go

```diff
@@ -18,53 +18,33 @@ package flags
 
 import (
 	"flag"
-	"fmt"
-	"log"
-	"regexp"
 	"strings"
 )
 
+const (
+	ALL     = "ALL"
+	SPANNER = "SPANNER"
+)
+
 // Avoid making these vars public.
 var (
-	changedFiles string
+	modulesToBuild string
+	moduleMap      = map[string]string{ALL: "", SPANNER: "v2/datastream-to-spanner/,v2/spanner-common/,v2/spanner-change-streams-to-sharded-file-sink/,v2/gcs-to-sourcedb/,v2/spanner-migrations-sdk/,v2/spanner-custom-shard/,v2/sourcedb-to-spanner/,v2/common/,metadata"}
```

Comment:
What about v1 spanner templates?

Reply:
Also, there have been issues in the past with not specifying `plugins/templates-maven-plugin` since that plugin is not explicitly depended on, but is needed for staging templates for IT's

Reply:
This is a good callout - @manitgupta probably makes sense to add v1 in here.

@Polber the plan here is also to evolve this over time once we add the spanner workflow, I'll add the plugins/templates-maven-plugin dep, but it is also known that this isn't quite an exhaustive list. @manitgupta will have a follow up PR to control this once we have the main infra in.

Reply:
In that case LGTM